src/main/java/com/ecommerce/Controller/ProductController.java

```diff
@@ -0,0 +1,98 @@
+package com.ecommerce.Controller;
+
+import com.ecommerce.Request.ProductRequest;
+import com.ecommerce.Service.ProductsService;
+import com.ecommerce.Service.ProductsServiceImpl;
+import com.ecommerce.Entity.Products;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1")
```

Comment:
all product endpoints must start with /products

Reply:
updated this, see commit e3f325d